macrosAkka/src/main/scala/com/softwaremill/macwire/macwireakka/MacwireAkkaMacros.scala

```diff
@@ -0,0 +1,14 @@
+package com.softwaremill.macwire
+package macwireakka
```

Comment:
Maybe we could simplify the package name to just `com.softwaremill.macwire.akka`? Not sure if there's any sense in repeating the macwire part